### DISCUSSION TITLE
voiranime.com: nuisance

### DISCUSSION THREAD

**Discussion by Ever4engel** [uBlockOrigin/uAssets#24775] [Q&A] [answered]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://v5.voiranime.com`

### Category

nuisance

### Description

This is a streaming website, everything work just fine with Ub0 on ads side, but seem this website use a captcha for every reader loaded (images will be provided in screenshots section).

the question is: it is possible to overide all those captcha chalenges?

### Other extensions used

Decentraleyes
Dark reader

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/user-attachments/assets/0a8c15f6-a5b0-4e16-9945-a4495e73d291)
![image](https://github.com/user-attachments/assets/b86138da-8c5c-4db4-a0df-bfb0473e68b1)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Firefox: 128
filterset (summary):
 network: 174234
 cosmetic: 37482
 scriptlet: 20737
 html: 1900
listset (total-discarded, last-updated):
 added:
  ara-0: 2332-262, 11m
 default:
  user-filters: 0-0, never
  ublock-filters: 39712-315, 1h.7m Δ
  ublock-badware: 10213-4, 1h.7m Δ
  ublock-privacy: 1039-5, 1h.7m Δ
  ublock-unbreak: 2529-2, 1h.7m Δ
  ublock-quick-fixes: 136-2, 1h.7m Δ
  easylist: 87794-14278, 1h.7m Δ
  easyprivacy: 53067-63, 1h.7m Δ
  urlhaus-1: 28735-0, 11m
  plowe-0: 3547-988, 11m
  FRA-0: 24766-3361, 11m
filterset (user): [empty]
userSettings:
 cloudStorageEnabled: true
 ignoreGenericCosmeticFilters: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 721 ms (selfie)
 maxAssetCacheWait: 101 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 7
 network:
  bidgear.com: 2
  blondparcelcane.com: 1
  exdynsrv.com: 2
  googletagmanager.com: 1
  realsrvcdn.com: 1
 extended:
  ##.ad
  ##+js(set-constant, rwt, noopFunc)
```
</details>

**Comment by D4niloMR** [comment, marked as answer]
> the question is: it is possible to overide all those captcha chalenges?

Not possible. A request must be made with the captcha token.
